plugins/circleci/src/hooks/useStepOutput.test.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { renderHook, waitFor } from '@testing-library/react';
+import { useApi } from '@backstage/core-plugin-api';
+import { useStepOutput } from './useStepOutput';
+
+jest.mock('@backstage/core-plugin-api', () => ({
+  ...jest.requireActual('@backstage/core-plugin-api'),
+  useApi: jest.fn(),
+}));
+
+jest.mock('./useProjectSlugFromEntity', () => ({
+  useProjectSlugFromEntity: () => ({ projectSlug: 'github/my-org/dummy' }),
```

Comment:
same here, just provide the actual entity in a wrapper with an `<EntityProvider>` instead perhaps?

Reply:
Fixed